src/js/themes/base.js

```diff
@@ -1935,6 +1935,24 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         margin: 'none',
       },
     },
+    toggleGroup: {
+      button: {
+        // iconOnly: {
+        //   pad: undefined,
+        // },
+        pad: {
+          horizontal: '11px',
```

Comment:
@halocline @britt6612 Final item to decide on for base theme, what should the default horizontal width be?

Here, it's the same for with label or icon only and looks like this:
<img width="532" alt="Screen Shot 2024-04-19 at 11 32 48 AM" src="https://github.com/grommet/grommet/assets/12522275/504c848f-4984-4c9b-8a29-da0a729f70b3">

Alternately, if we aligned to the larger horizontal pad for default buttons it would have a value of `23px` and look like this (where iconOnly ones would still retain their 11px):

<img width="626" alt="Screen Shot 2024-04-19 at 11 34 57 AM" src="https://github.com/grommet/grommet/assets/12522275/8d1d8a14-dae9-4482-9732-a0970ff23983">

I was questioning the larger horizontal pad because that is geared more towards buttons with full rounding. Let me know which value you both are good with.

Reply:
I agree 
`that the larger horizontal pad because that is geared more towards buttons with full rounding`

I'd like the smaller option.

Reply:
Probably the smaller

Reply:
Great, then ready for final review.